### PR TITLE
add basic vulnerability reference system

### PR DESCRIPTION
**Pull request by vdeturckheim**
Adding a referencing system for our vulnerabilities.

WDYT @nodejs/security-wg ?

### CODE REVIEW

**Comment by ChALkeR** [comment]
Mostly great! A question — should this specify preference in case if both an entry here and CVE exist?

**Comment by vdeturckheim** [comment]
@ChALkeR basically, CVEs and NSWGs are not in a 1-1 relationship, for instance NSWG-COR-7 (https://github.com/nodejs/security-wg/blob/master/vuln/core/7.json) is linked to "CVE-2016-9840.",
```
[
    "CVE-2016-9840.",
    "CVE-2016-9841.",
    "CVE-2016-9842.",
    "CVE-2016-9843."
  ]
```

**Comment by lirantal** [comment]
This makes sense so that we have our own security advisories system kind of like the big linux distros do (RHSA, USN etc).

Where and for which reasons should we prefer to use the NSWG reference as opposed to the CVE one?

File: vuln/REAMDE.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Node.js Core and Ecosystem vulnerabilities
+
+This directory contains two list of disclosed vulnerabilities:
```
Comment (mhdawson) [inline vuln/REAMDE.md]: two list**s**

**Comment by vdeturckheim** [comment]
@mhdawson I don't have any strong opinion on that atm. Right no the difference is mostly historical an could be removed in the future.

**Pull request merged**